### PR TITLE
Creating project.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: mysql/tram-saga-schema.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
+USE eventuate;
+
+DROP Table IF Exists aggregate_instance_subscriptions;
```
Comment (cer) [inline mysql/tram-saga-schema.sql]: The base image is `eventuateio/eventuate-tram-mysql:0.3.0.RELEASE`. It should contain this schema.

File: integration-tests/src/test/resources/embedded-schema.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
+DROP Table IF Exists message;
```
Comment (cer) [inline integration-tests/src/test/resources/embedded-schema.sql]: There should be no need to duplicate this schema.
The scripts that you need should be on the classpath, e.g.  https://github.com/eventuate-tram/eventuate-tram-core/blob/master/eventuate-tram-in-memory/src/main/resources/eventuate-tram-embedded-schema.sql and https://github.com/eventuate-tram/eventuate-tram-sagas/blob/master/eventuate-tram-sagas-in-memory/src/main/resources/eventuate-tram-sagas-embedded.sql

File: mysql/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+FROM eventuateio/eventuate-tram-mysql:0.3.0.RELEASE
```
Comment (cer) [inline mysql/Dockerfile]: I suspect that you don't need to build an image for the project. There is no custom schema

File: set-env-mysql.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
+if [ -z "$DOCKER_HOST_IP" ] ; then
```
Comment (cer) [inline set-env-mysql.sh]: Move the duplicate code in set-env-*.sh into a set-env.sh that both of them `. ./set-env.sh`

File: order-common/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,3 @@
+FROM java:openjdk-8u111-alpine
```
Comment (cer) [inline order-common/Dockerfile]: Why is this here!!!

File: integration-tests/src/test/resources/embedded-schema.sql (hunk 2 above)
Comment (dartartem) [reply integration-tests/src/test/resources/embedded-schema.sql]: ok

File: mysql/Dockerfile (hunk 3 above)
Comment (dartartem) [reply mysql/Dockerfile]: I will check

File: mysql/tram-saga-schema.sql (hunk 1 above)
Comment (dartartem) [reply mysql/tram-saga-schema.sql]: I will check

File: order-common/Dockerfile (hunk 5 above)
Comment (dartartem) [reply order-common/Dockerfile]: Sorry, bad copy/paste, I will fix it.

File: set-env-mysql.sh (hunk 4 above)
Comment (dartartem) [reply set-env-mysql.sh]: no problem
